### PR TITLE
Add Autocode

### PR DESCRIPTION
**Pull request by mrhappyma**
hopefully i did this right...

### CODE REVIEW

File: entries/a/autocode.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Autocode": {
+    "domain": "autocode.com",
+    "img": "autocode.com.png",
```
Comment (Carlgo11) [inline entries/a/autocode.com.json]: SVG-files are prefered over PNGs. I suggest you use this one:

```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/a/autocode.com.json (hunk 1 above)
Comment (mrhappyma) [reply entries/a/autocode.com.json]: Oh thanks, I wasn't able to find an svg.

**Comment by Carlgo11** [comment]
One of our tests won't complete and I suspect the bug is because you, the commit author, isn't registered on CircleCI.

I'll make a bug report to CircleCI but in the meantime if you could either:
* Log in to [CircleCI](circleci.com) with your GitHub account.
* Allow maintainer commits to your PR
  

Either of those actions would cause the build to succeed.

Thanks and sorry for the inconvenience.

**Review by Carlgo11 — APPROVED** [review]

**Comment by mrhappyma** [comment]
Alrighty, i've logged into circleci.com with my github account.

**Comment by Carlgo11** [comment]
Thanks! Everything works now.

**Review by phallobst — APPROVED** [review]

**Pull request merged**